examples/recursive-proofs/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "license": "UNLICENSED",
+  "scripts": {
+    "clean": "rm -rf build",
+    "compile": "yarn clean && tsc",
+    "start": "node build/main.js",
+    "clean:codegen": "rm -rf codegen",
+    "export:leaf": "nargo export --program-dir=./circuits/sum",
+    "export:recurseLeaf": "nargo export --program-dir=./circuits/recurseLeaf",
+    "export:recurseNode": "nargo export --program-dir=./circuits/recurseNode",
+    "export:all": "yarn clean:codegen && yarn export:leaf && yarn export:recurseLeaf && yarn export:recurseNode",
```

Comment:
Looks like we don't use noir_codegen here?

Reply:
Evolved from another example, and did remove use of codegen artifacts.